tests/automation_framework/data/work_order/work_order_submit_encryptedDataEncryptionKey_empty_echoClient.json

```diff
@@ -0,0 +1,37 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkOrderSubmit",
+    "id": 11,
+    "params": {
+        "responseTimeoutMSecs": 6000,
+        "payloadFormat": "json-rpc",
+        "resultUri": "resulturi",
+        "notifyUri": "notifyuri",
+        "workOrderId": "",
+        "workerId":"",
+        "workloadId": "echo-result",
```

Comment:
workload_Id is echo_result but the Data parameters are for heart disease eval.

Reply:
Thanks for the review. As workload id is echo_result its normal functionality is echo the available data. So provided alphanumeric data with spaces. But to avoid confusion changing to "Hello world" string.